packet/packet.go

```diff
@@ -710,6 +710,20 @@ func (packet *Packet) GetPacketSegmentLen() uint {
 	return low.GetDataLenMbuf(packet.CMbuf)
 }
 
+// GetPacketPayload returns extracted packet payload as byte array and bool status.
+// Works only for protocols, supported by ParseData (IPv4, IPv6, TCP, UDP, ICMP)
+func (packet *Packet) GetPacketPayload() ([]byte, bool) {
+	pktLen := packet.GetPacketSegmentLen()
+	pktStartAddr := packet.StartAtOffset(0)
+	pktBytes := (*[1 << 30]byte)(pktStartAddr)[:pktLen]
```

Comment:
AAAAA magic number, magic number AAAAA. To be serious, we have GetRawPacketBytes function which is not zero-copy as yours one. You should use it for pktBytes and also mention about zero-copy in comment.